native/mjml_nif/Cargo.toml

```diff
@@ -11,7 +11,10 @@ crate-type = ["cdylib"]
 
 [dependencies]
 rustler = "0.33.0"
-mrml = { version = "3.1", default-features = false, features = ["parse", "render", "orderedmap"] }
+mrml = { version = "4.0", default-features = false, features = [
+    "parse",
+    "render",
+] }
```

Comment:
Thanks @moomerman for the PR, very much appretiated :pray:.

What's the reason for removing the `"oderedmap"` feature? I saw that it's enabled by default since mrml 2.0, but since we don't use all default features (`default-features = false`) I think we need to keep it in order to make sure the order of the created html is consistent (see #63)?

Reply:
I'm don't fully understand it, but it fails to compile with `orderedmap` in:

```
cargo build
warning: `/Users/richard/workspace/moomerman/mjml_nif/native/mjml_nif/.cargo/config` is deprecated in favor of `config.toml`
note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
error: failed to select a version for `mrml`.
    ... required by package `mjml_nif v4.0.0 (/Users/richard/workspace/moomerman/mjml_nif/native/mjml_nif)`
versions that meet the requirements `^4.0` (locked to 4.0.0) are: 4.0.0

the package `mjml_nif` depends on `mrml`, with features: `orderedmap` but `mrml` does not have these features.


failed to select a version for `mrml` which could resolve this conflict
```

It look like it was [removed upstream](https://github.com/jdrouet/mrml/commit/f9cdfd07d06a976cdea996279bb82e436028007b).

Reply:
Ah, yes, you are right of course: mrml only uses indexmap (i.e. the former orderemap feature) from 4.0 on and I did not see it, when I first scrolled  through the changelog: https://github.com/jdrouet/mrml/blob/87bec4bddcd3b9601d1f06b67e8b996512ed9e07/packages/mrml-core/CHANGELOG.md?plain=1#L38

So I'll merge your changes and prepare a new release later today. Thanks again! :purple_heart: